packages/next/build/swc/src/styled_jsx/transform_css.rs

```diff
@@ -121,16 +137,31 @@ impl Namespacer {
         // One off global selector
         if &name.value == "global" {
           let block_tokens = get_block_tokens(&args);
+          let mut front_tokens = get_front_selector_tokens(&args);
           let mut args = args.clone();
-          args.tokens.extend(block_tokens);
-          let complex_selectors: Vec<ComplexSelector> = parse_tokens(
-            &args,
-            ParserConfig {
-              parse_values: false,
-            },
-          )
-          .unwrap();
-          return complex_selectors[0].selectors.clone();
+          front_tokens.extend(args.tokens);
+          front_tokens.extend(block_tokens);
+          args.tokens = front_tokens;
+          let complex_selectors = panic::catch_unwind(|| {
```

Comment:
I think this will print something to console and it can be replaced with a match statement.